### PR TITLE
ck8s:Fix cluster.local in kubeconfigs

### PR DESCRIPTION
**Pull request by lucianvlad**
**What this PR does / why we need it**:
Fix cluster.local in kubeconfigs
**Which issue this PR fixes** *(use the format `fixes #<issue number>(, fixes #<issue_number>, ...)` to automatically close the issue when PR gets merged)*: fixes #

**Public facing documentation PR** *(if applicable)*

**Special notes for reviewer**:

**Checklist:**

- [x] Proper commit message prefix on all commits
- [ ] Updated the [public facing documentation](https://github.com/elastisys/compliantkubernetes)
- Is this changeset backwards compatible for existing clusters? Applying:
  - [ ] is completely transparent, will not impact the workload in any way.
  - [ ] requires running a migration script.
  - [ ] requires draining and/or replacing nodes.
  - [ ] will break the cluster.
        I.e. full cluster migration is required.

### CODE REVIEW

File: migration/v2.19.x-ck8sx-v2.20.x-ck8s1/upgrade-cluster.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+1. Switch to the correct remote: `git submodule sync`
+
+1. Update the kubespray submodule: `git submodule update --init --recursive`
+
+1. set the values for `kubeconfig_cluster_name` in both `sc-config/group_vars/k8s_cluster/ck8s-k8s-cluster.yaml` and `wc-config/group_vars/k8s_cluster/ck8s-k8s-cluster.yaml` to the coresponding name like bellow:
+
+    ```yaml
+    kubeconfig_cluster_name: <CHANGE-ME-ENVIRONMENT-NAME-sc>
+    kubeconfig_cluster_name: <CHANGE-ME-ENVIRONMENT-NAME-wc>
+    ```
```
Comment (viktor-f) [inline migration/v2.19.x-ck8sx-v2.20.x-ck8s1/upgrade-cluster.md:14]: The `upgrade-cluster` part is not needed for your change, but we will want it in this doc later anyway.
```suggestion
1. set the values for `kubeconfig_cluster_name` in both `sc-config/group_vars/k8s_cluster/ck8s-k8s-cluster.yaml` and `wc-config/group_vars/k8s_cluster/ck8s-k8s-cluster.yaml` to the coresponding name like bellow:

    ```yaml
    kubeconfig_cluster_name: <CHANGE-ME-ENVIRONMENT-NAME-sc>
    kubeconfig_cluster_name: <CHANGE-ME-ENVIRONMENT-NAME-wc>
    ```
1. Upgrade the cluster to a new kubernetes version:

    ```bash
    ./bin/ck8s-kubespray run-playbook sc upgrade-cluster.yml -b
    ./bin/ck8s-kubespray run-playbook wc upgrade-cluster.yml -b
    ```

1. generate updated kubeconfigs:

    ```bash
    ./bin/ck8s-kubespray run-playbook sc ../playbooks/kubeconfig.yml -b
    ./bin/ck8s-kubespray run-playbook wc ../playbooks/kubeconfig.yml -b
    ```
```

**Review by Xartos — APPROVED** [review]
:+1: Do we also want to create a patch release for `2.19`?

File: migration/v2.19.x-ck8sx-v2.20.x-ck8s1/upgrade-cluster.md (hunk 1 above)
Comment (lucianvlad) [reply migration/v2.19.x-ck8sx-v2.20.x-ck8s1/upgrade-cluster.md:14]: done

**Review by viktor-f — APPROVED** [review]
If we want we could create a patch, although I think it can wait until the next kubespray release.

File: migration/v2.19.x-ck8sx-v2.20.x-ck8s1/upgrade-cluster.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# Upgrade v2.19.x-ck8sx to v2.20.x-ck8s1
+
+1. Checkout the new release: `git checkout v2.20.x-ck8s1`
+
+1. Switch to the correct remote: `git submodule sync`
+
+1. Update the kubespray submodule: `git submodule update --init --recursive`
+
+1. set the values for `kubeconfig_cluster_name` in both `sc-config/group_vars/k8s_cluster/ck8s-k8s-cluster.yaml` and `wc-config/group_vars/k8s_cluster/ck8s-k8s-cluster.yaml` to the coresponding name like bellow:
```
Comment (OlleLarsson) [inline migration/v2.19.x-ck8sx-v2.20.x-ck8s1/upgrade-cluster.md:9]:
```suggestion
1. set the values for `kubeconfig_cluster_name` in both `sc-config/group_vars/k8s_cluster/ck8s-k8s-cluster.yaml` and `wc-config/group_vars/k8s_cluster/ck8s-k8s-cluster.yaml` to the corresponding name like below:
```

**Review by OlleLarsson — APPROVED** [review]

File: migration/v2.19.x-ck8sx-v2.20.x-ck8s1/upgrade-cluster.md (hunk 2 above)
Comment (lucianvlad) [reply migration/v2.19.x-ck8sx-v2.20.x-ck8s1/upgrade-cluster.md]: fixed
thx

**Pull request merged**